Sources/Exporters/DatadogExporter/Persistence/FilesOrchestrator.swift

```diff
@@ -113,6 +113,16 @@ internal class FilesOrchestrator {
         }
     }
 
+    func getAllFiles(excludingFilesNamed excludedFileNames: Set<String> = []) -> [ReadableFile]? {
+        do {
+            return try directory.files()
+                .filter({ excludedFileNames.contains($0.name) == false })
+        } catch {
+            print("🔥 Failed to obtain readable files: \(error)")
```

Comment:
is it typical practice to log with emojis? Interesting...

Reply:
It is on swift, since it has always had full unicode support, and in fact is quite useful for visualising errors in a long log